include/openssl/ssl.h

```diff
@@ -218,6 +218,7 @@ extern "C" {
 # define SSL_TXT_aPSK            "aPSK"
 # define SSL_TXT_aGOST94 "aGOST94"
 # define SSL_TXT_aGOST01 "aGOST01"
+# define SSL_TXT_aGOST12 "aGOST12"
 # define SSL_TXT_aGOST  "aGOST"
```

Comment:
Something strange going on with the alignment here (the strings are out of alignment with all the others in this file). Or did github just mess this up?